### ISSUE TITLE
Google Consent broken

### ISSUE THREAD

**Issue by U-Go-7** [AdguardTeam/AdguardBrowserExtension#1824]
### Prerequisites
win 10
firefox 89.0.1 freshly installed or right after deleting cache and cookies.
Adguard Steath Mode active with "Hide Referrer from third-parties" on.

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x ] I am running the latest version
- [x ] I checked the documentation and found no answer
- [x ] I checked to make sure that this issue has not already been filed

### Issue Details

- **AdGuard version:**
3.6.6
- **Browser and version:**
firefox 89.0.1
- **Operating system and version:**
Windows 10
- **Filters you use in AdGuard:**
all but custom
- **Stealth Mode settings:**
enabled
- **Other browser extensions:**
irrelevant (tested)

### Expected Behavior

when the browser is new or right after deleting cache and cookies, the first time searching something in google a popup will appear asking to give consent to search customization, youtube history and ad personalization. It should be possible to select all "off" and confirm and be redirected to the wished search.

### Actual Behavior

error message "There seems to be a problem. Please make your choice again.". I am forced to give consent.

### Screenshots

![adguard](https://user-images.githubusercontent.com/69869741/122647661-a3fa0a00-d125-11eb-8440-9e12a46e84a9.png)

<details><summary>Screenshot:</summary>

</details>

### Additional Information

I have made lot of tests. Glad I did because for a long time I blamed ClearURLs.
Testing with no other addon, with any possible firefox privacy setting, the result is always the SAME: when the option " Hide Referrer from third-parties" is active, consent in that google popup is forced. When the option is disabled, it's possible to choose "off" in that google consent.

It is of the utmost importance for me to say (fuck) off to anything google, so I am forced to deactivate this option of yours, which is actually inconvenient because I did wish to have it active.

Thanks

**Comment by anastenko** [comment]
@U-Go-7 Hello.
We couldn't reproduce it on our end. Maybe there are some other conditions?

**Comment by U-Go-7** [comment]
> We couldn't reproduce it on our end. Maybe there are some other conditions?

I guess you mean that you do not even get the Google consent thing at all?
Or you get the consent but not the error message when sending your preferences?

If you do not get the consent, it's because of local laws.
I am in Germany.
When I use a US IP, I also do not get the consent.
But whenever I get the consent, I get  the error message.

**Comment by anastenko** [comment]
@U-Go-7 Indeed, the problem was location-based. I've managed to reproduce it. 
We will take a look.

**Comment by anastenko** [comment]
@U-Go-7 This issue should be fixed with the next version 4.0.

**Comment by U-Go-7** [comment]
Nice, thanks :)
